packagehandlers/gradlepackagehandler.go

```diff
@@ -61,7 +61,7 @@ func (gph *GradlePackageHandler) updateDirectDependency(vulnDetails *utils.Vulne
 	isAnyDescriptorFileChanged := false
 	for _, descriptorFilePath := range descriptorFilesPaths {
 		var isFileChanged bool
-		isFileChanged, err = fixVulnerabilityIfExists(descriptorFilePath, vulnDetails)
+		isFileChanged, err = fixGradleVulnerabilityIfExists(descriptorFilePath, vulnDetails)
```

Comment:
why we need the word gradle?
its already gradle's file

Reply:
because we have a similar function with a similar name for nuget and they are in the same package so I wanter to differentiate the names